include/fast_float/float_common.h

```diff
@@ -78,6 +78,28 @@
 #endif
 #endif
 
+#ifdef FASTFLOAT_SSE2
```

Comment:
Above, we would like something like this...

```C++
#if !defined(FASTFLOAT_SSE2) && (defined(__x86_64)  || defined(__x86_64__) || defined(_M_X64) || defined(__amd64) || defined(_M_AMD64))
#define FASTFLOAT_SSE2
#endif
```

Or something of the sort so that SSE2 support is picked up automagically.